package.json

```diff
@@ -1,34 +1,43 @@
 {
-	"name": "@rbxts/replicaservice",
-	"version": "1.0.0-ts.7",
-	"description": "ReplicaService types",
-	"main": "src/init.lua",
-	"homepage": "https://github.com/Mixu78/replicaservice-typings#readme",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/Mixu78/replicaservice-typings"
-	},
-	"scripts": {},
-	"keywords": [],
-	"author": "",
-	"license": "ISC",
-	"types": "src/index.d.ts",
-	"files": [
-		"src"
-	],
-	"publishConfig": {
-		"access": "public"
-	},
-	"devDependencies": {
-		"@rbxts/compiler-types": "^2.0.4-types.0",
-		"@rbxts/types": "^1.0.642",
-		"@typescript-eslint/eslint-plugin": "^5.28.0",
-		"@typescript-eslint/parser": "^5.28.0",
-		"eslint": "^8.17.0",
-		"eslint-config-prettier": "^8.5.0",
-		"eslint-plugin-prettier": "^4.0.0",
-		"eslint-plugin-roblox-ts": "^0.0.32",
-		"prettier": "^2.7.0",
-		"typescript": "^4.8.3"
-	}
+  "name": "@rbxts/replicaservice",
+  "version": "1.0.0-ts.8",
```

Comment:
You seem to have updated the ReplicaService lua code as well, the minor version should be bumped instead of type version

Reply:
Wouldn't it be better if both type and minor version were increased?

Reply:
I have increased the minor version by 1, I'll be waiting for your answer of the last comment.

Reply:
Type version should be reset to 0 as this is a new minor version.